### DISCUSSION TITLE
Billing on Github Campus Program - Enterprise Account

### DISCUSSION THREAD

**Discussion by randiproska** [community/community#65109] [Enterprise]
### Hi everyone,

Hello everyone,

I'm Randi Proska, and I am affiliated with a university that is a proud member of the GitHub Campus Program. I have come across information stating that Github Copilot is available at no cost for GitHub Campus Partners. However, upon reviewing my enterprise account, I noticed a calculated charge associated with the monthly usage of GitHub Copilot. I am wondering whether this is to be expected or if there's a requirement for me to make payments for this usage. Your clarification on this matter would be greatly appreciated.

Best regards,

**Comment by arafajs** [comment]
- GitHub offers a variety of benefits to Campus Partners, including free private repositories, GitHub Pages, and GitHub Classroom. However, **GitHub Copilot is not currently one of these benefits.**
- **You have to** sign up for GitHub Copilot separately and link your GitHub account that has the Student Developer Pack.

**Comment by LeandroLuna** [comment]
Hi, randiproska!

You can access GitHub Copilot for free by using the [GitHub Student Developer Pack](https://education.github.com/pack?WT.mc_id=academic-97170-cyzanon). Since you're enrolled in a university, perhaps you can unlock many benefits over there.

I'm personally utilizing Copilot through this method.

If you're experimenting any unwanted billing, I suggest to you redo Copilot activation in your account.

**Comment by randiproska** [comment]
Hi @arafajs and @LeandroLuna, I am also registered as a lecturer in my institution. Can I have access for Github Copilot?

**Comment by randiproska** [comment]
Hi, @arafajs I thought that by joining GitHub Campus Partners, we would receive a free enterprise account. Does this mean that GitHub Copilot is not included?

**Reply by arafajs** [reply]
1. Verify your teaching status on GitHub Global Campus. You can do this by going to the GitHub Global Campus website: https://education.github.com/ and clicking on the "Verify Teaching Status" button.
2. Once your teaching status has been verified, you will be automatically notified about your free GitHub Copilot subscription. You can check your eligibility by going to the GitHub Copilot subscription page: https://github.com/settings/copilot.
3. If you are eligible, you will be able to activate your free GitHub Copilot subscription by clicking on the "Activate" button.

**Reply by randiproska** [reply]
Hi @arafajs,

I am confused about why I cannot reverify my teaching status, even though I use my institutional email address. The message consistently states, "We require applicants to use a school-issued email address to apply". However, I have already utilized my institutional email address and correctly selected my institution's name in the "What is the name of your school?" field. Additionally, I am the administrator for my institution's Campus Partner program.

Can you help me with this? Thanks

**Reply by arafajs** [reply]
It seems that there might be a problem with your GitHub account or your institution's registration in GitHub Education.
I think you should Contact GitHub Education support for help.

**Reply by randiproska** [reply]
Hi @arafajs. Actually, I ended up here because I received no response from GitHub after submitting a complaint through a ticket to GitHub Education support. I almost never get any response from them, some of my previous tickets haven't received a response for over 5 months
